imap/message.c

```diff
@@ -985,6 +985,77 @@ static int read_headers_condstore_qresync_updates(struct ImapAccountData *adata,
 
   return 0;
 }
+
+/**
+ * imap_verify_qresync - Check to see if QRESYNC got jumbled. 
+ * If so, wipe the context and try again with a normal download. 
+ * @param m  Imap Selected Mailbox
+ *
+ * @retval  0 Success
+ * @retval -1 Error
+ */
+// TODO refactor this
```

Comment:
All the data we need for this function is in the `Mailbox`, or its private mdata `ImapMboxData`.
Data that was Account-level in Mutt, is now Mailbox-level.
The single `ctx` in Mutt is just the `m` Mailbox in NeoMutt.
Oh, and a bool name has been swapped for its opposite -- I'll leave you to discover which :-)